### PR TITLE
Completed exercises

### PR DESCRIPTION
**Pull request by levibrooke**

### CODE REVIEW

File: js/letters.js [hunk 1]
```diff
@@ -15,8 +15,24 @@ var counter = {
...
+  if (text.length === 0) {
+    return counter;
+  }
+
+  // convert string to lowercase
+  let newText = text.toLowerCase();
+
+  let targetChar = newText.slice(0, 1);
+
+  if ([targetChar]) {
```
Comment (taesup) [inline js/letters.js]: The square brackets are not necessary in this situation and will actually always return true here. 

You'll also need to figure out if the particular character you're getting is in the counter object or not. Check out the object.hasOwnProperty method for a way to double check if a key exists in an object.

**Review by taesup — CHANGES_REQUESTED** [review]

File: js/letters.js (hunk 1 above)
Comment (levibrooke) [reply js/letters.js]: So this instead? This is passing the test.

```javascript
if (counter.hasOwnProperty(targetChar)) {
```

**Comment by levibrooke** [comment]
Updated letters.js to reflect change on line 30.

**Comment by taesup** [comment]
LGTM!

**Pull request merged**